novawallet/Modules/Vote/Crowdloan/CrowdloanList/CrowdloanListInteractor.swift

```diff
@@ -471,3 +476,16 @@ extension CrowdloanListInteractor: SelectedCurrencyDepending {
         }
     }
 }
+
+extension CrowdloanListInteractor: EventVisitorProtocol {
+    func processNetworkEnableChanged(event: NetworkEnabledChanged) {
+        guard
+            let chain = crowdloanState.settings.value,
+            chain.chainId == event.chainId
+        else  {
+            return
+        }
+        
+        refresh(with: chain)
```

Comment:
The problem here is that CrowdloanChainSettings is not updating automatically. What we need to do is to call crowdloanState.settings.setup(...) and then refresh in callback with provided chain.